icu4c/source/i18n/unicode/numberrangeformatter.h

```diff
@@ -612,8 +613,9 @@ class U_I18N_API LocalizedNumberRangeFormatter
     ~LocalizedNumberRangeFormatter();
 
   private:
-    // TODO: This is not thread-safe! Do NOT check this in without an atomic here.
```

Comment:
^ oops. well, it's corrected now..

Reply:
Yeah, I expected this change to require more discussion so I saved it for a follow-up PR.